get_info.php

```diff
@@ -1 +1,308 @@
-<?php?>
+<?php
+require 'get_query_generators.php';
+header("Content-type: application/json");
+try {
+    $method = "";
+    if (isset($_SERVER['REQUEST_METHOD'])) {
+        $query = "";
+        $method = $_SERVER['REQUEST_METHOD'];
+        // Request stored as associative array
+        $data = json_decode(file_get_contents("php://input"), true);
+        switch ($method) {
+            case "GET":
+                print json_encode(handle_get($_GET));
+                break;
+
+            default:
+                throw new Exception("Invalid Request");
+                break;
+        }
+    } else {
+        $error = 'No Request Method Found';
+        throw new Exception($error);
+    }
+} catch (Exception $e) {
+    $result = set_http_response(400);
+    date_default_timezone_set('America/Toronto');
+    error_log(
+        date("Y-m-d h:i:sa") . " : " . $e->getMessage() . "\n",
+        3,
+        "errors.log"
+    );
```

Comment:
error logging should be removed once we're ready for production,  add as comment.